### DISCUSSION TITLE
8월 29일 : 월말평가 피드백

### DISCUSSION THREAD

**Discussion by numerical43** [cca-ffodregamdi/running-hi-back#199] [Ideas]
### 발표 자료 : [8월_월말평가_러닝하이.pdf](https://github.com/cca-ffodregamdi/running-hi-back/files/12462143/8._._.pdf)

* 기획의도 ㄱㅊ

* 깃 브랜치 전략은 단순히 5개로 나누는것만이 아님. 현재는 release, hotfix를 사용하고 있지않으니까 나중에 배포 후에 설명하기

* 스토리북 사용한 캡쳐 넣으면 더 좋을 듯

* 컨텍스트 api는 ?? 딜링을 막기위해 쓴거다
스테이트를 관리할 목적은 아님 -> 리덕스 쓰거나 유즈스테이트 써야함

* aws 파일 업로드
에디터 이미지 비동기 방식으로 올린 뒤 글 작성하면 함께 저장하는 방식일 때 이미지 업로드 후 뒤로가기 하면 사진은 이미 업로드 된 상태. 수정 필요함 -> 처음엔 temp 폴더에 저장했다가 글을 저장하면 기본 폴더로 옮기기. 만약에 중간에 해당 페이지를 나가거나 글을 저장하지않으면 temp 폴더에 남아있음. 매일 스케쥴러를 돌려서 temp 폴더 비우기.

* Spring Config
레파지토리에 비밀번호 암호화된 값 넣기

* 토큰 시간 암호화 안해도 됨
csrf.disable 하면 안됨

* 시그니쳐 관리 어케 했는지 보여주면 더 좋을듯

* 로그인 시나리오
리프레쉬 토큰은 보안을 포기하겠다는 말이다!!

* 컨벤션
pk uuid 득실이 확실함
사용한 이유가 확실하게 있어야 함
한계 이상의 사용자가 들어오면 어떻게 할건지??

* 레코드 하나 이용하기 위해서 17버전으로 올렸다는 것은 많은 것을 포기했을 수 있다. 레코드 안쓰고 불변객체 구현하는 게 낫다!
버전을 올린 더 확실한 이유가 필요하다.

* GPX Parser
거리가 먼 경우에는 필요하지만 사람이 뛸 수 있는 거리인 경우 호의 길이까지 필요 없을수도
3초 사이에 지그재그로 뛰었을 때 생기는 오차가 더 크다. 타임을 짧게 가져가면서 부하 줄일 수 있는 방법 찾기

**Comment by Subak-Uncle** [comment]
정리 감사합니다!
